src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -85,6 +85,7 @@ def train(self, runner):
 
         """
         last_return = None
+        runner.enable_logging = False
```

Comment:
hmm this seems like a foot-gun, especially because we don't explicitly test logging right now.